lib/web_apis/browser_api.es6.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+foam.CLASS({
+  name: 'BrowserAPI',
+  package: 'org.chromium.apis.web',
+  documentation: `BrowserAPI is a FOAM model that contains browser name,
+  	browser version, OS, OS version, interface name and API name.`,
+  properties: [
```

Comment:
Please add `documentation: ...` for each property. In particular, `interfaceName` and `apiName` need to be clearly documented as being `<interface-name>(.prototype)?.<api-name>` from Javascript bindings.